### PR TITLE
Feat/list

### PR DESCRIPTION
**Pull request by Vikas2426**
image
image

### CODE REVIEW

**Review by jeetsdev — APPROVED** [review]
Everything looks good to me.

File: list/style.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
+@import url("../global.css");
+
+body {
+  padding: 2rem;
+}
```
Comment (manismk) [inline list/style.css]: Avoid direct tag styling in component style sheets.

**Review by manismk — APPROVED** [review]
Looks good

File: list/style.css (hunk 1 above)
Comment (Vikas2426) [reply list/style.css]: removed

**Pull request merged**